tests/src/test/scala/org/apache/spark/sql/rapids/execution/InternalColumnarRDDConverterSuite.scala

```diff
@@ -0,0 +1,60 @@
+package org.apache.spark.sql.rapids.execution
+
+import com.nvidia.spark.rapids.{ColumnarToRowIterator, GpuBatchUtilsSuite, NoopMetric, SparkQueryCompareTestSuite}
+import com.nvidia.spark.rapids.GpuColumnVector.GpuColumnarBatchBuilder
+
+import org.apache.spark.sql.types.{BinaryType, DoubleType, StructField, StructType}
+
+class InternalColumnarRDDConverterSuite extends SparkQueryCompareTestSuite {
+  test("transform binary data back and forth between Row and Columnar") {
```

Comment:
Hi @jlowe , when running this unit test from maven:
```bash
mvn test -Dbuildver=312 -Pspark312tests -DwildcardSuites="org.apache.spark.sql.rapids.execution.InternalColumnarRDDConverterSuite"
```
the following error is thrown: 
```bash
*** RUN ABORTED ***
  java.lang.UnsatisfiedLinkError: ai.rapids.cudf.ColumnView.getNativeValidPointerSize(I)J
  at ai.rapids.cudf.ColumnView.getNativeValidPointerSize(Native Method)
  at ai.rapids.cudf.HostColumnVector$ColumnBuilder.growBuffersAndRows(HostColumnVector.java:988)
  at ai.rapids.cudf.HostColumnVector$ColumnBuilder.appendNull(HostColumnVector.java:1024)
  at org.apache.spark.sql.rapids.execution.GpuExternalRowToColumnConverter$BinaryConverter$.append(InternalColumnarRddConverter.scala:351)
  at org.apache.spark.sql.rapids.execution.GpuExternalRowToColumnConverter.$anonfun$convert$1(InternalColumnarRddConverter.scala:45)
  at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:158)
  at org.apache.spark.sql.rapids.execution.GpuExternalRowToColumnConverter.convert(InternalColumnarRddConverter.scala:44)
  at org.apache.spark.sql.rapids.execution.InternalColumnarRDDConverterSuite.$anonfun$new$3(InternalColumnarRDDConverterSuite.scala:17)
  at org.apache.spark.sql.rapids.execution.InternalColumnarRDDConverterSuite.$anonfun$new$3$adapted(InternalColumnarRDDConverterSuite.scala:17)
  at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:36)

```
I checked the cuDF jar used by maven (remove all cuDF jars in .m2/repository/ before run the test), and can see the libcudf.so contains this method by grep.

Since this test is copied from [GpuColumnarToRowSuite.scala](https://github.com/NVIDIA/spark-rapids/blob/branch-22.02/tests/src/test/scala/com/nvidia/spark/rapids/GpuColumnarToRowSuite.scala#L48), but there's no error when I run 
```bash
 mvn test -Dbuildver=312 -Pspark312tests -DwildcardSuites="com.nvidia.spark.rapids.GpuColumnarToRowSuite"
```
I think both tests are running through the call to `getNativeValidPointerSize`. 
Could you help guide some directions here? Thank you!